npm/azure-npm.yaml

```diff
@@ -172,6 +172,7 @@ data:
             "EnableV2NPM":             true,
             "PlaceAzureChainFirst":    false,
             "ApplyInBackground":       true,
-            "NetPolInBackground":      true
+            "NetPolInBackground":      true,
+            "EnableNPMLite":           true
```

Comment:
Dont change this file, instead create a new one

Reply:
created a new one!